apps/profile/admin.py

```diff
@@ -66,7 +59,7 @@ def get_edit_link(self, instance):
 
         return html.format_html("""<a href="{url}">{text}</a>""".format(
                 url=url,
-                text="Ändere Bild %s auf Seperaten Seite" % instance.profileimage._meta.verbose_name,
+                text="Change %s here" % instance.profileimage._meta.verbose_name,
```

Comment:
Wie das eigentlich gemeint ist, findest du im Profiel Model bei der __str__ methode.

Reply:
merge das mal 
ich mach das mit dem phil im nächsten pull request für die übersetzung rein
weil jetzt bei der übersetzung der fuzzy fehler ist 
muss das mit dem phil machen der kann das etwas besser als ich 

local bei mir steht da jetzt  _(Change %s here) % ....

Reply:
@TUmadao Mach mal bitte daraus _('Changes %(image_name)s here.') % {'image_name': instance.profileimage._meta.verbose_name} 

Das dann an den anderen Stellen auch. Der Philipe kann das dann bei Zeiten in Deutsch übersetzten.

Reply:
ist gepusht